src/renderer/entities/transaction/ui/Deposit/Deposit.test.tsx

```diff
@@ -4,8 +4,6 @@ import { act, render, screen } from '@testing-library/react';
 import { Deposit } from './Deposit';
 import type { Asset } from '@shared/core';
 
-jest.mock('@renderer/components/common');
-
 jest.mock('@app/providers', () => ({
```

Comment:
Maybe try to remove this mock and check it again? Because main problem was is wrong dependencies, and now it may be fixed